colin-api/src/colin_api/resources/v1/corporations.py

```diff
@@ -120,6 +122,42 @@ def request_colin(corp_num: str):  # pylint: disable=too-many-locals, too-many-b
     return jsonify(corp_details_dict), 200
 
 
+@bp.route('/business/<string:corp_num>', methods=['GET', 'OPTIONS'])
+@cross_origin(origin='*')
+def business_request_colin(corp_num: str):
+    """Get business details from COLIN."""
+    corp_num_sql = "\'" + corp_num + "\'"
+    incorp_info_sql = Methods.build_incorp_info_sql(corp_num_sql)
+
+    try:
+        business_info_dict = nro.get_business_info_by_corp_num(corp_num=corp_num_sql)
```

Comment:
` business_info_dict = nro.get_business_info_by_corp_num(corp_num=corp_num_sql)` does not work.  

Passing in "corp_num" insstead of "corp_num_sql" does.

Reply:
I did